media-libs/mesa/mesa-9999.ebuild

```diff
@@ -226,7 +227,7 @@ pkg_pretend() {
 		   ! use video_cards_radeonsi &&
 		   ! use video_cards_v3d &&
 		   ! use video_cards_virgl; then
```

Comment:
I think we forgot to actually add asahi to this conditional?

Reply:
indeed. I thought I did an asahi only build but apparently not. thanks